addons/docs/src/frameworks/ember/jsondoc.js

```diff
@@ -26,5 +26,10 @@ export const extractProps = componentName => {
 export const extractComponentDescription = componentName => {
   const json = getJSONDoc();
   const componentDoc = json.included.find(doc => doc.attributes.name === componentName);
+
+  if (!componentDoc) {
```

Comment:
I've discovered that sometimes the parameter.component isn't there, and then this code throws because componentDoc is `undefined`